svthalia/concrexit
# Make price serializers match PaymentAmountField

@TippieCodes ran into an issue with the API crashing on serializing certain fields, apparently a bogus high fine.

I found out that this is because the serializer has a lower `max_digits` than `PaymentAmountField`.

I fixed this both for fines, and the other `DecimalField` serializers I could find that were related to `PaymentAmountField`s.




### How to test
Steps to test the changes you made:
1. Create an event with a huge fine (6 digits or more)
2. Open the event list in the API
3. Observe that it does not crash
